ui/src/components/Header.tsx

```diff
@@ -38,14 +42,14 @@ export default function Header() {
       }}
     >
       <Toolbar>
-        <Mobile />
+        <Mobile onBrandClick={onBrandClick} />
         <Desktop />
       </Toolbar>
     </AppBar>
   );
 }
 
-function Mobile(props: { window?: () => Window }) {
+function Mobile(props: { window?: () => Window; onBrandClick?: () => void }) {
```

Comment:
I feel like it's nicer to just destructure in the function definition, or at the very least to destructure in the first line of the function. That way it's clear what the props are, and I don't see a need for passing along all the props as a single object. So why not something like this instead?

```suggestion
function Mobile({window, onBrandClick}: { window?: () => Window; onBrandClick?: () => void }) {
```

Reply:
Sounds good, updated